documentation/Team-roles-guidelines.md

```diff
@@ -0,0 +1,70 @@
+Working in teams in this project team members will have different roles. Scrum Master, Quality Assurance (QA), and Developer play distinct yet interconnected roles. Below are guidelines for each role:
+
+### Scrum Master(SM):
+
+1. **Facilitator and Leader:**
+   - Act as a facilitator for the Scrum Team, removing impediments and ensuring a conducive environment for development.
+   - Serve as a servant leader, putting the needs of the team first and helping them achieve their goals.
+
+2. **Process Guardian:**
+   - Ensure that the Scrum framework is understood and followed by the team.
+   - Facilitate Scrum events, such as Sprint Planning, Daily Stand-ups, Sprint Review, and Sprint Retrospective.
+
+3. **Coordinator:**
+   - Promote effective communication within the team and acts as a link between teams. 
+   - Address conflicts and foster collaboration among team members.
+   - Promote collaboration between teams, Scrum masters meeting such as Sprint planning and spring retrospective.
+   - Establish a dedicated channel for Scrum Masters to collaborate and communicate effectively.
+
+4. **Continuous Improvement:**
+   - Encourage the team to regularly reflect on their processes and identify areas for improvement.
+   - Help the team adapt to change and continuously refine their working practices.
+
+### Quality Assurance (QA):
+
+1. **Responsability:**
+   - Quality Assurance serves as the primary reviewer for the project, ensuring thorough assessment and validation of deliverables.
+   - Review the code changes and provide feedback.QA may verify that the reported issue has been adequately addressed and resolved. 
+   - QA is responsible for merging, and if the issue is properly linked, it should automatically close itself. However, it sometimes encounters bugs.
+   - Ensure that quality is maintained throughout the development process.
+   - Collaborate with developers to understand user stories and acceptance criteria.
+   - Establish a dedicated channel for Quality Assurances to collaborate and communicate effectively.
```

Comment:
```suggestion
   - Establish a dedicated channel for all Quality Assurance teams to collaborate and communicate effectively.
```

Reply:
maybe better: channel for QAs from all teams